proxy/coco/src/peer.rs

```diff
@@ -1,19 +1,12 @@
 //! Machinery to advance the underlying network protocol and manage auxiliary tasks ensuring
 //! prorper state updates.
 
-use std::{
-    future::Future,
-    io,
-    net::SocketAddr,
-    pin::Pin,
-    task::{Context, Poll},
-    vec,
-};
+use std::{io, net::SocketAddr, vec};
 
-use futures::{future::FutureExt as _, stream::StreamExt as _};
+use futures::prelude::*;
```

Comment:
Is the wildcard import for the entire prelude necessary?

Reply:
Not really. If you want I can import the traits explicitly.

Reply:
It seems preferrable to be explicit esp. with pulling in the traits.